mongoengine/fields.py

```diff
@@ -20,11 +21,6 @@
 else:
     import dateutil.parser
 
-try:
```

Comment:
I got rid of this as I don't see how bson.int64 wouldn't be importable. Maybe this was addressing a problem in ancient versions of pymongo?!